### ISSUE TITLE
Has this been deprecated?

### ISSUE THREAD

**Issue by Znegl** [EddyVerbruggen/nativescript-plugin-firebase#1624]
We're currently on the hunt for a QR scanner for use in a NativeScript/Vue app and this seems like a perfect fit, but when looking at the Firebase documentation for "ML Kit for Firebase" it seems like the SDK has been deprecated and split into two new SDK's: "Firebase ML" and "ML Kit".

Can anyone clarify if this repository relies on the deprecated SDK or if it's just the documentation that hasn't been updated to reflect the changes?

We would prefer not to set up a Firebase account (along with all of the legal and GDPR stuff).

**Comment by EddyVerbruggen** [comment]
Can you share a link to the docs you're referring to?

Anyway: if there's a new SDK then we're using the older one.

**Comment by Znegl** [comment]
Yes of course! I forgot, but here it is: https://firebase.google.com/docs/ml-kit

**Comment by EddyVerbruggen** [comment]
Ah, nice... more work.

Tagging this as a feature request to be able to use the standalone on-device ML Kit features.

**Comment by Znegl** [comment]
I'm sorry to bring such annoying news 🙈

**Comment by cjohn001** [comment]
Hello together, I am also trying to get the QR code scanning working with mlkit. I tried to get the demo app running at https://github.com/EddyVerbruggen/nativescript-plugin-firebase/tree/master/demo-ng. However, have not had success with it yet. Maybe I need to setup an account with firebase first to get the demo running? But actually I am posting to the feature request here as I also only need the mlkit based on device QR code scanner functionality. Would be great if I could keep all the firebase stuff out of my app. I am therefore wondering if this feature can be expected for the near future?
